specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2020-04-01-preview/EventGrid.json

```diff
@@ -0,0 +1,3798 @@
+          "Topics_Delete": {
+            "$ref": "./examples/Topics_Delete.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "patch": {
+        "tags": [
+          "Topics"
+        ],
+        "summary": "Update a topic.",
+        "description": "Asynchronously updates a topic with the specified parameters.",
+        "operationId": "Topics_Update",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "description": "The name of the resource group within the user's subscription.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "topicName",
+            "in": "path",
+            "description": "Name of the topic.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "topicUpdateParameters",
+            "in": "body",
+            "description": "Topic update information.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/TopicUpdateParameters"
+            }
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Topic update request accepted.",
+            "schema": {
+              "$ref": "#/definitions/Topic"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 Bad Request.\n\n * 500 Internal Server Error."
+          }
+        },
+        "x-ms-examples": {
+          "Topics_Update": {
+            "$ref": "./examples/Topics_Update.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.EventGrid/topics": {
+      "get": {
+        "tags": [
+          "Topics"
+        ],
+        "summary": "List topics under an Azure subscription.",
+        "description": "List all the topics under an Azure subscription.",
+        "operationId": "Topics_ListBySubscription",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/FilterParameter"
+          },
+          {
+            "$ref": "#/parameters/TopParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/TopicsListResult
```

Comment:
Are their other values? If not, does this need to be exposed?

Reply:
this is intentional. May expand few more during GA version  of this feature.